lib/trento/infrastructure/operations/operations.ex

```diff
@@ -46,5 +46,7 @@ defmodule Trento.Infrastructure.Operations do
   end
 
   def map_operation_type("saptuneapplysolution@v1"), do: :saptune_solution_apply
+  def map_operation_type("saptunechangesolution@v1"), do: :saptune_solution_change
+
```

Comment:
nitpick: i would remove this new line (if it is not added by the formatter)

Reply:
13c6ee8c722373a1d1f0a246246eb656f14fb2b2